package.json

```
@@ -8,21 +8,66 @@
     "build-prod": "cross-env NODE_ENV=production webpack",
     "dev": "cross-env NODE_ENV=development webpack-dev-server --open",
     "prod": "npm-run-all --parallel start-prod-server build-prod",
-    "test": "jest"
+    "test": "jest",
+    "e2e-test": "cypress run -b chrome",
+    "e2e-test:open": "cypress open"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "src/client/components/**"
```

```suggestion
      "src/client/**"
```

вдруг ты захочешь какой-нибудь сервис создать или тот же redux